src/components/results/ResultTableComponent.wc.svelte

```diff
@@ -122,9 +122,18 @@
     interface Props {
         title?: string;
         pageSize?: number;
+        showPageSize?: boolean;
     }
 
-    let { title = "", pageSize = 10 }: Props = $props();
+    /**
+     *  Configuration options for the result table.
+     * title:  The title to be displayed in the table header.
+     * pagesize: The number of rows to display per page. Defaults to 10.
+     * showPageSize: Whether to display a select box for changing the pagesize. If you set the pagesize otherthen 10,25,50 it will added as an option.
```

Comment:
Better move this to the individual prop definitions in the interface. That way you will get IntelliSense for the props when using the component.